searxng/searxng
# simple theme: move the source files to searx/statics/themes/simple/src

## What does this PR do?

See commit messages.
Directory structure of searx/static/themes/simple :
* src : contains source code <--- so the source is in one location.
* css / fonts / js: contains build files (include css/images )
* img : contains image for this theme

The dependencies 
* has been updated (build
* autocomplete-js and leaflet are declared as packages.json depdenencies.

`devDependencies` of `packages.json` includes all required dependencies, so  
https://github.com/searxng/searxng/blob/127552f67bfa8d19f43cd8d77a256e525633c5a7/manage#L16
is not required.

## Why is this change important?

* Same clean up than the oscar theme
* moving the source code to another direction will be more clear ( https://github.com/searxng/searxng/issues/141 ).

## How to test this PR locally?

1. clean browser cache
1. using the simple theme:
  * check autocompletion:
    * type text, press enter
    * type text, choose an autocompletion, press enter
  * check the images rendering
  * check "show map" of the OSM / photon results.
  * check various icons are displayed correctly (generated by webfont plugin).

## Author's checklist



## Related issues

Related to https://github.com/searxng/searxng/issues/141
